examples/shell/shell_common/cmd_ping.cpp

```diff
@@ -19,13 +19,16 @@
 #include <stdio.h>
 #include <stdlib.h>
 
-#include <core/CHIPCore.h>
+#include <lib/core/CHIPCore.h>
+
+#if CONFIG_DEVICE_LAYER
```

Comment:
why this #if? 

Could we separate this and add the code as a conditional compiled file?

Reply:
I think we can remove this flag since no command except help available w/o device layer and all platforms defined this flag